packages/stylelint-config-airbnb/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "stylelint-config-airbnb",
+  "version": "0.0.0",
+  "description": "Airbnb's stylelint config, following our styleguide",
+  "main": "index.js",
+  "scripts": {
+    "lint": "eslint .",
+    "test": "npm run lint"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/airbnb/css.git"
+  },
+  "keywords": [
+    "airbnb",
+    "css",
+    "lint",
+    "stylelint",
+    "stylelint-config",
+    "stylelint-config-airbnb"
+  ],
+  "author": "Stephen Edgar",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/airbnb/css/issues"
+  },
+  "homepage": "https://github.com/airbnb/css#readme",
+  "devDependencies": {
+    "eslint": "^2.11.1",
+    "eslint-config-airbnb": "^9.0.1",
+    "eslint-plugin-import": "^1.8.1",
+    "eslint-plugin-jsx-a11y": "^1.2.3",
+    "eslint-plugin-react": "^5.1.1",
+    "stylelint": "^6.5.1",
```

Comment:
Right - what I mean is, this config is likely compatible with specific versions of stylelint - making it a peer dep allows us to convey "which versions we're compatible with" in a way that fits easily with npm.

If a package wouldn't ever reasonably be installed without X, and it requires X to work, then X must be a dep or a peer dep :-)